clang/docs/ReleaseNotes.rst

```diff
@@ -340,6 +340,9 @@ Bug Fixes to C++ Support
   by template argument deduction.
 - Clang is now better at instantiating the function definition after its use inside
   of a constexpr lambda. (#GH125747)
+- Clang no longer crashes when trying to unify the types of arrays with
+  certain differences in qualifiers, per ternary operator or template argument
+  deduction. (#GH97005)
```

Comment:
```suggestion
- Clang no longer crashes when trying to unify the types of arrays with
  certain differences in qualifiers (this could happend during template argument
  deduction or when building a ternary operator). (#GH97005)
```